### PR TITLE
Nvli 333 mirador ui

### PR DESCRIPTION
**Pull request by safallia-joseph**
Recreated the "Resource" feature from dev.

### CODE REVIEW

File: docroot/modules/features/resource/config/install/core.entity_view_display.node.resource.default.yml [hunk 1]
```diff
@@ -58,9 +58,10 @@ content:
   field_rating:
     weight: 3
     label: above
-    settings: {  }
+    settings:
+      fivestar_widget: modules/fivestar/widgets/basic/basic.css
     third_party_settings: {  }
-    type: fivestar_rating
+    type: fivestar_stars
```
Comment (piyuesh23) [inline docroot/modules/features/resource/config/install/core.entity_view_display.node.resource.default.yml]: This doesn't look related.

File: docroot/modules/features/resource/config/optional/views.view.mirador_view.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ dependencies:
     - core.entity_view_mode.node.mirador
     - node.type.resource
     - user.role.administrator
+    - user.role.anonymous
```
Comment (piyuesh23) [inline docroot/modules/features/resource/config/optional/views.view.mirador_view.yml]: Change the view access to permission based. Users having access to view published content should be able to access mirador view.

File: docroot/modules/features/resource/config/install/core.entity_view_display.node.resource.default.yml (hunk 1 above)
Comment (safallia-joseph) [inline docroot/modules/features/resource/config/install/core.entity_view_display.node.resource.default.yml]: yeah! I recreated the "Resource" feature from dev. It had this much changes.
Maybe somebody directly modified in dev, and did not exported as feature.
Or change made in code and not synced in dev!

File: docroot/modules/features/resource/config/optional/views.view.mirador_view.yml (hunk 2 above)
Comment (safallia-joseph) [inline docroot/modules/features/resource/config/optional/views.view.mirador_view.yml]: ok

File: docroot/modules/features/resource/config/install/core.entity_view_display.node.resource.default.yml (hunk 1 above)
Comment (safallia-joseph) [inline docroot/modules/features/resource/config/install/core.entity_view_display.node.resource.default.yml]: @piyuesh23 removed it in https://github.com/NVLI/nvli-main-website/pull/88/commits/6fe67993996517923a46670110dcc6df68842154

File: docroot/modules/features/resource/config/optional/views.view.mirador_view.yml (hunk 2 above)
Comment (safallia-joseph) [inline docroot/modules/features/resource/config/optional/views.view.mirador_view.yml]: modified it in https://github.com/NVLI/nvli-main-website/pull/88/commits/6fe67993996517923a46670110dcc6df68842154
